newswires/client/src/WireDetail.tsx

```diff
@@ -19,50 +22,67 @@ export const WireDetail = ({ wire }: { wire: WireData }) => {
 			: undefined;
 	}, [wire]);
 
-	const listItems = [
-		{
-			title: 'Byline',
-			description: byline ?? 'Not found',
-		},
-		{
-			title: 'Keywords',
-			description: (
-				<EuiFlexGroup wrap responsive={false} gutterSize="xs">
-					{keywords?.map((keyword) => (
-						<EuiFlexItem key={keyword} grow={false}>
-							<EuiBadge color="primary">{keyword}</EuiBadge>
-						</EuiFlexItem>
-					))}
-				</EuiFlexGroup>
-			),
-		},
-		{
-			title: 'Usage restrictions',
-			description: usage ?? 'Not found',
-		},
-		{
-			title: 'Body text',
-			description: safeBodyText ? (
-				<article dangerouslySetInnerHTML={{ __html: safeBodyText }} />
-			) : (
-				'Not found'
-			),
-		},
-	];
+	const nonEmptyKeywords = useMemo(
+		() => keywords?.filter((keyword) => keyword.trim().length > 0) ?? [],
+		[keywords],
+	);
 
 	return (
 		<Fragment>
-			<h3
-				css={css`
-					font-weight: 300;
-				`}
-			>
-				{wire.content.subhead}
-			</h3>
+			<EuiScreenReaderLive focusRegionOnTextChange>
+				<h3
+					css={css`
+						font-weight: 300;
+					`}
+				>
+					{wire.content.subhead}
+				</h3>
+			</EuiScreenReaderLive>
 
 			<EuiSpacer size="m" />
 
-			<EuiDescriptionList listItems={listItems} />
+			<EuiDescriptionList>
```

Comment:
pass-by refactor -- now we're getting a lot more wires we're getting quite a few entries that are missing fields, or which have `keywords: [""]`, so doing some checks before rendering empty headings makes sense.

can split out into a separate PR if wished